CMakeLists.txt

```diff
@@ -83,7 +83,7 @@ if (DO_PLATFORM_LINUX)
     # (Enable --trace-expand in cmake and look at lines from boost_headers-config.cmake and its callers.)
     # Workaround: force cmake to look in "/usr" since the /lib and /bin paths are either
     # not relevant for our usage or they are symlinks to /usr/lib and /usr/bin.
-    set(CMAKE_PREFIX_PATH "/usr")
+    # set(CMAKE_PREFIX_PATH "/usr")
```

Comment:
As I mentioned in the other comment, let us set CMAKE_PREFIX_PATH only if it is empty here. Once changed, make sure you verify that overriding from build.py and using the default works properly.